src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers } from 'redux';
+import booksReducer from './books/books';
+import catReducer from './categories/categories';
+
+const reducer = combineReducers({
+  booksRed: booksReducer,
+  catRed: catReducer,
+});
+
+const store = createStore(
+  reducer(),
+);
+
+export default store;
```

Comment:
- [ ] **[OPTIONAL]** What do you think about Importing `applyMiddleware` - to use some handy Redux middleware, like a redux logger. This will help you in the development process. To install it, just run:

  ```
  npm i --save redux-logger
  ```